components/eventing-controller/pkg/handlers/eventtype/clean_test.go

```diff
@@ -25,60 +26,65 @@ func TestCleaner(t *testing.T) {
 			givenApplicationName: "testapp",
 			givenEventType:       "sap.kyma.testapp.order.created.v1",
 			wantEventType:        "sap.kyma.testapp.order.created.v1",
+			wantError:            false,
```

Comment:
What about adding tests for Clean() with empty application list?

Reply:
Done